invenio_github/models.py

```diff
@@ -238,6 +238,9 @@ def enabled(self):
 
     def latest_release(self, status=None):
         """Chronologically latest published release of the repository."""
+        # Bail out fast if object not in DB session.
+        if self not in db.session:
```

Comment:
If we do a session detection then we should do it correctly with the `inspect` method.